src/OrchardCore/OrchardCore.Search.AzureAI.Core/Services/AzureAISearchIndexingService.cs

```diff
@@ -73,11 +73,6 @@ public async Task ProcessContentItemsAsync(params string[] indexNames)
             lastTaskId = Math.Min(lastTaskId, taskId);
         }
 
-        if (indexSettings.Count == 0)
```

Comment:
@MikeAlhayek this check is always `false` because of

https://github.com/OrchardCMS/OrchardCore/blob/c1b4cb897ee738f9fa3f8ac3f13d5156febc3456/src/OrchardCore/OrchardCore.Search.AzureAI.Core/Services/AzureAISearchIndexingService.cs#L64-L67

Reply:
No. What if the user passed invalid index name. `ProcessContentItemsAsync("Some Bad IndexName")` in this case this check will protect you.

Reply:
The problem is not the first check, please have a look at the line 60 - 80 you will see the same check twice, the second one is useless because the first one protect us

Reply:
I see. so remove line 76-79 and leave 64-67